gapis/api/vulkan/frame_loop.go

```diff
@@ -144,8 +144,8 @@ type frameLoop struct {
 	eventChanged   map[VkEvent]bool
 	eventDestroyed map[VkEvent]bool
 
-	framebuffersToDestroy map[VkFramebuffer]bool
-	framebuffersToCreate  map[VkFramebuffer]bool
+	framebufferToDestroy map[VkFramebuffer]bool
```

Comment:
I'm wondering why moving from plural to singular on this variable? Still a list of framebuffers, right?

Reply:
Just for uniformity with the other maps. It was easier to move the outlier into the group than move the group to the outlier.